apps/portal/modules/config.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2016 WSO2, Inc. (http://wso2.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var getConfigFile;
+
+(function () {
+    getConfigFile = function () {
+        var carbon = require('carbon');
+        try {
+            return carbon.server.loadConfig('dashboards/portal.json');
+        } catch (e) {
+            log.error("Config " + e);
```

Comment:
Add more context in the error message. The given error message doesn't provide any information.